content/60.methods.md

```diff
@@ -1,50 +1,101 @@
-The volume of information available, much of which has not gone through rigorous peer review, presents a significant challenge to individual efforts to keep abreast of the state of COVID-19 research [@doi:10.1038/s42254-020-0175-7].
-However, research on these topics is proceeding so rapidly that any static review is likely to quickly become dated.
-Our goal as a community is to consolidate information about the virus in the context of related viruses and to synthesize rapidly emerging literature centered on the diagnosis and treatment of COVID-19.
-We used an open publishing framework, Manubot [@doi:10.1371/journal.pcbi.1007128], to manage hundreds of contributions from the community to create a living, scholarly document.
+Coronavirus Disease 2019 (COVID-19) has shaped the years 2020 and 2021 by causing a world-wide public health crisis.
+The scientific community has responded by turning significant attention and resources towards COVID-19 and the associated virus, SARS-CoV-2.
+The result has been the rapid release of data, results, and publications related to COVID-19 at a scale never previously seen.
+Over 20,000 articles about COVID-19 were released in the first 4 months of the pandemic [@doi:10.1053/j.ackd.2020.08.003].
+The velocity and volume of information being released led to the pandemic being termed as an "infodemic" as well [@doi:10.1053/j.ackd.2020.08.003; @doi:10/ggpx67].
+While this influx of information is likely evidence of important work towards understanding the virus and the disease, there are also downsides to the availability of too much information.
+The potential for "excessive publication" has been identified as an issue for over forty years, and was one concern about the move towards electronic, rather than print, publishing at the turn of the millennium [@doi:10/d3bmnv].
+<!--To Do: Flesh out-->
+
+<!-- To Do: This is just an outline, formalize this paragraph with citations etc-->
+While some of this information has been disseminated by traditional publishing mechanisms, in other cases, it is made public through preprint servers or even press releases.
+Preprints provide a venue for scientists to release findings rapidly, but have both the advantage and disadvantage of making research available before it has undergone the peer review process.
+Media outlets don't always report on this accurately.
+Additionally, many papers are being retracted.
+These include both preprints and papers that are published in more traditional venues.
+The large number of retractions may also be influenced by the fact that the time from submission to peer review for papers related to COVID-19 is very low.
+
+The rate of this proliferation also presents challenges to efforts to summarize and synthesize existing literature, which are necessary given the volume.
```

Comment:
This is an important point to emphasize here an maybe elsewhere.  There have been related efforts to review COVID-19 preprints (which we should search for an cite).  We can explain why our project goes beyond reviewing preprints and the advantages of summarizing and synthesizing a massive amount of preprints, news stories, and journal publications.

Reply:
That sounds great! I can definitely add this.